maxmind_consts.py

```diff
@@ -47,3 +47,5 @@
 MMDB_FILE = "GeoLite2-City.mmdb"
 MMDB_TAR_FILE = "GeoLite2-City.tar.gz"
 DB_DOWNLOAD_URL = "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key={0}&suffix=tar.gz"
+
+DEFAULT_REQUEST_TIMEOUT = 30  # in seconds
```

Comment:
I'm using the same timeout value as this app https://github.com/splunk-soar-connectors/forescoutcounteract/blame/2d63a3fd8c39c4e7d069e34022c08e5e85928820/forescoutcounteract_consts.py#L83